src/elm/Session/LoggedIn.elm

```diff
@@ -293,7 +293,7 @@ viewHelper thisMsg page profile_ ({ shared } as model) content =
 
             Hidden ->
                 text ""
-        , div [ class "flex-grow" ]
+        , div [ class "flex-grow bg-white xl:bg-gray-500" ]
```

Comment:
why add this? The app looks weird:

<img width="1764" alt="Screen Shot 2020-07-30 at 13 30 49" src="https://user-images.githubusercontent.com/1082127/88948898-e3e3a480-d268-11ea-9775-b7903dcc34e1.png">

Reply:
Please disregard it. It was a tailwind testing that went through. It was not supposed to be committed.